scss/components/button.scss

```diff
@@ -625,3 +635,7 @@ $block: #{$fd-namespace}-button;
     }
 
 }
+.fd-div--dropdown {
```

Comment:
we can avoid this class completely by using `fd-popover` here. 

also `div` is not a good choice for class name as we don't reference HTML elements.